packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.test.js

```diff
@@ -208,8 +208,136 @@ describe('EnumerationWidget', () => {
 			expect(toJson(wrapper)).toMatchSnapshot();
 		});
 
+
+		it('should send a event with a method to simulate the click on the input file', () => {
+			// given
+			const registry = {
+				formContext: {
+					handleAction: jest.fn(),
+				},
+			};
+
+			const wrapper = mount(
+				<EnumerationWidget
+					registry={registry}
+					schema={{
+						allowImport: true,
+					}}
+				/>);
+
+			// when
+			wrapper.find('.tc-enumeration-header').find('.btn-link').at(1).simulate('click');
+
+			// then
+			expect(registry.formContext.handleAction).toBeCalledWith(
+				undefined,
+				'ENUMERATION_IMPORT_FILE_CLICK',
+				{
+					simulateClickInputFile: jasmine.any(Function),
+				},
+				jasmine.any(Function),
+				jasmine.any(Function),
+			);
+		});
+
+		it('should send a event with a method to simulate the click on the input file', () => {
+			// given
+			const registry = {
+				formContext: {
+					handleAction: jest.fn(),
+				},
+			};
+
+			const wrapper = mount(
+				<EnumerationWidget
+					registry={registry}
+					schema={{
+						allowImport: true,
+					}}
+				/>);
+
+			// when
+			wrapper
+				.find('.tc-enumeration-header div.btn-group-link button').at(0)
+				.simulate('click');
+
+			// then
+			expect(registry.formContext.handleAction).toBeCalledWith(
+				undefined,
+				'ENUMERATION_IMPORT_FILE_CLICK',
+				{
+					simulateClickInputFile: jasmine.any(Function),
+				},
+				jasmine.any(Function),
+				jasmine.any(Function),
+			);
+		});
+
+		it('should send a event with the choice done by the user', () => {
```

Comment:
instead of something generic like this, can you change it so the expect is more specific ? (we need to read the test to know what it does)
in that case the aim is to send an import file event in append mode.